src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -65,4 +65,33 @@ public Entity getEntityFromDatastore(String entityName, String entityPropertyTit
     Entity entity = resultsList.get(0);
     return entity;
   }
+
+   
+   /**
+    * Returns a single Entity object that can then be used in a servlet. 
+    * The Entity is retrieved from matching an entity property w a cookie on the server. 
+    * <p>
+    * This method always returns immediately, whether or not the 
+    * datastore object exists. If the object does not exist, the entity 
+    * returned is null.
+    * 
+    * @param  cookies    an array of cookies (usually all cookies on front end)
+    * @param  datastore  the database where entities are stored
+    * @return            a single entity that has the cookie name/value pair as a property
+    */
+ 
+  public Entity getUserFromCookie(Cookie cookies[], DatastoreService datastore){
+      if(datastore == null){
+          return null;
```

Comment:
I'd log this, you can see the screen shot I sent in the chat it should be exactly the same